packages/lit-html/src/lit-html.ts

```diff
@@ -755,6 +767,12 @@ class TemplateInstance {
     this._template = template;
   }
 
+  _disconnect() {
```

Comment:
Let's add a comment describing what this does and make especially clear that it recurses down the tree of parts that have disconnect

Reply:
Done